io/net/irqbalance.py

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Shaik Abdulla <shaik.abdulla1@ibm.com>
+
+'''
+Irq-balance and CPU affinity test for IO subsystem.
+'''
+
+import re
+import netifaces
+from avocado import Test
+from avocado.utils import process, cpu, wait, dmesg, genio
+from avocado.utils.network.interfaces import NetworkInterface
+from avocado.utils.network.hosts import LocalHost
+from avocado.utils.process import CmdError
+import multiprocessing
+import subprocess
+import time
+
+totalcpus = int(multiprocessing.cpu_count()) - 1
+errorlog = ['WARNING: CPU:', 'Oops',
+            'Segfault', 'soft lockup', 'ard LOCKUP',
+            'Unable to handle paging request',
+            'rcu_sched detected stalls',
+            'NMI backtrace for cpu',
+            'Call Trace:']
```

Comment:
can you please add "input/output error" here?
so that on irqbalance affinity change, we can ignore if this error occures

Reply:
@Naresh-ibm  as discussed, the "input/output error" are not logging/captured  in dmesg, there would be no use adding of this lines here.